source/MRMesh/MRUnorientedTriangle.h

```diff
@@ -9,21 +9,23 @@ namespace MR
 
 /// describes a triangle as three vertex IDs sorted in a way to quickly find same triangles
 /// irrespective of vertex order (clockwise or counterclockwise)
-struct UnorientedTriangle : ThreeVertIds
+struct UnorientedTriangle
```

Comment:
why?

Reply:
Maybe we can make it work, but inheriting from standard containers doesn't look like good practice. :thinking: